lib/mutant/cache.rb

```diff
@@ -21,7 +24,7 @@ def initialize
     # @api private
     def parse(path)
       @cache.fetch(path) do
-        @cache[path] = Parser::CurrentRuby.parse(File.read(path))
+        @cache[path] = Parser::CurrentRuby.parse(file.read(path))
```

Comment:
> Pathname does not have Pathname.read(a_file) interface.

No, but you can use `path.read`. This code will become:

```ruby
def parse(path)
  @cache.fetch_or_store(path) do |path|
    Parser::CurrentRuby.parse(path.read)
  end
end
```

The expectation you'd have to add to the specs will be even simpler than the current iteration.